src/core/issues-notes.js

```diff
@@ -241,10 +241,16 @@ function isLight(rgb) {
  */
 function createLabelsGroup(labels, title, repoURL) {
   const labelsGroup = labels.map(label => createLabel(label, repoURL));
+  let labelNames = labels.map(label => label.name);
```

Comment:
please see my previous comments. Try not to switch data types (i.e., always use `const`).